apps/jscad-web/static/index.html

```diff
@@ -82,6 +82,7 @@ <h2>Options</h2>
             <li><label for="dark-mode">Dark Mode</label><input type="checkbox" id="dark-mode"></li>
             <li><label for="show-axis">Show Axis</label><input type="checkbox" id="show-axis" checked></li>
             <li><label for="show-grid">Show Grid</label><input type="checkbox" id="show-grid" checked></li>
+            <li><label for="show-grid">Smooth render</label><input type="checkbox" id="smooth-render" checked></li>
```

Comment:
```suggestion
            <li><label for="smooth-render">Smooth Render</label><input type="checkbox" id="smooth-render" checked></li>
```

Clicking on the label was checking the box for "show grid" instead of Smooth Render. Also changed capitalization to be consistent with the other menu items.